catalog/transformer/catalog-transformer-pptx/src/main/java/ddf/catalog/transformer/input/pptx/PptxInputTransformer.java

```diff
@@ -111,7 +111,11 @@ private Metacard transformLogic(InputStream input)
       Metacard metacard =
           extractInitialMetadata(fileBackedOutputStream.asByteSource().openStream());
 
-      extractThumbnail(metacard, fileBackedOutputStream.asByteSource().openStream());
+      try {
+        extractThumbnail(metacard, fileBackedOutputStream.asByteSource().openStream());
+      } catch (IllegalStateException e) {
```

Comment:
Since the above code actually throws EncryptedDocumentException, I think we should reduce that catch down to that exception instead. We should also document in the javadoc for the extractThumbnail() method that it can throw EncryptedDocumentException.